Gaming the Syllabus/Assets/Scripts/Portal.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
```

Comment:
One thing you can consider is to use [RequireComponent(Collider2D)] to make it so this script can only be attached if the object has a collider. Minor thing, but can help explain what the component does and avoid some mistakes in usage